File: Basic C++/Pattern Program/Pattern5.cpp
//Printing this pattern
// 1
// 22
// 333
// 4444
#include<iostream>
using namespace std;
int main(){
    int n,i=1;
    cin>>n;
    while (i<=n)
    {
        int j=1;
        while(j<=i)
        {
            cout<<i;
            j++;
        }
        cout<<endl;
        i++;
    }
    

}

File: Patterns javascript/pattern5.js
let pattern = "";
let n = 6;
for (let i = 1; i <= n; i++) {
  for (let j = 1; j <= i; j++) {
    pattern += " ";
  }
  for (let k = n; k >= i; k--) {
    pattern += " *";
  }
  pattern += "\n";
}

console.log(pattern);
